sql_generators/mobile_kpi_support_metrics/templates/new_profiles.schema.yaml

```diff
@@ -0,0 +1,65 @@
+fields:
+- mode: NULLABLE
+  name: first_seen_date
+  type: DATE
+  description: When the profile has sent a baseline ping for the first time.
```

Comment:
```suggestion
  description: Date we first received a baseline ping from the profile.
```

Reply:
unless this is our standard description, in which case i won't fight for the change.